google/cloud/firestore_v1/collection.py

```diff
@@ -375,13 +392,25 @@ def end_at(self, document_fields):
         return query.end_at(document_fields)
 
     def get(self, transaction=None):
-        """Deprecated alias for :meth:`stream`."""
-        warnings.warn(
-            "'Collection.get' is deprecated:  please use 'Collection.stream' instead.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        return self.stream(transaction=transaction)
+        """Read the documents in this collection.
```

Comment:
I might be missing something, but is this de-deprecation related to limit to last?

Reply:
Yes, we need to reverse result came from backend before returning it to user, so streaming is not our way, and `get()` method becomes the best choice. As for de-deprecating this method in `CollectionReference()`, see the other comment about methods matching:

> There is a test which checks that Query() has the same (or less) methods that CollectionReference() does. So, as I understood that, the list of the supported methods should be the same for Query() and CollectionReference(), at least in our case.